modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+[id="running-the-performance-creator-profile-cluster_{context}"]
+= Running the Performance Profile Creator using `podman`
+
+As a cluster administrator, you can run `podman` and Performance Profile Creator to create a performance profile.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare-metal hardware.
+* A node with `podman` and OpenShift CLI (`oc`) installed.
+
+
+.Procedure
+
+.  Run `must-gather` on your cluster:
++
+[source,terminal]
+
+----
+$ oc adm must-gather --image=<PAO image> --dest-dir=<dir>
+----
+
++
+.Example
++
+[source,terminal]
+----
+$ oc adm must-gather --image=quay.io/openshift-kni/performance-addon-operator-must-gather:4.8-snapshot --dest-dir=must-gather
+----
+. Check the machine config pool with the following command:
+
++
+[source,terminal]
+----
+$ oc get mcp
+----
+.Example output
++
+[source,terminal]
+----
+NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
+worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
+----
+
+.  (Optional) Display help for the PPC tool as follows:
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator quay.io/openshift-kni/performance-addon-operator:4.8-snapshot -- -h
```

Comment:
If you change this to `podman run --entrypoint performance-profile-creator quay.io/openshift-kni/performance-addon-operator:4.8-snapshot -h` we can avoid 
`Error: required flag(s) "mcp-name", "must-gather-dir-path", "reserved-cpu-count", "rt-kernel" not set` and we'll be able to see output as follows:

```
A tool that automates creation of Performance Profiles

Usage:
  performance-profile-creator [flags]

Flags:
      --disable-ht                        Disable Hyperthreading
  -h, --help                              help for performance-profile-creator
      --mcp-name string                   MCP name corresponding to the target machines (required) (default "worker-cnf")
      --must-gather-dir-path string       Must gather directory path (default "must-gather")
      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
      --profile-name string               Name of the performance profile to be created (default "performance")
      --reserved-cpu-count int            Number of reserved CPUs (required)
      --rt-kernel                         Enable Real Time Kernel (required) (default true)
      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
      --user-level-networking             Run with User level Networking(DPDK) enabled

```